build.gradle.kts

```diff
@@ -1,6 +1,6 @@
 plugins {
-  id("org.hypertrace.docker-plugin") version "0.5.1"
-  id("org.hypertrace.docker-publish-plugin") version "0.5.1"
+  id("org.hypertrace.docker-plugin") version "0.6.0"
```

Comment:
This will want to move to 0.6.1 for the same empty branch issue we saw in https://app.circleci.com/pipelines/github/hypertrace/helm-gcs-packager/29/workflows/aba305d1-6051-405a-a397-85a7b5b91b81/jobs/33

Reply:
ah good eye!

Reply:
I guess it technically doesn't matter unless we're making the circleci changes here too, since that's only encountered on a tag-triggered release.